.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: minimal
+
+dist: xenial
+
+addons:
+  apt:
+    packages:
+      - astyle
+      - perltidy
+      - python-pip
+
+install:
+  - sudo pip install yapf
+
+script:
+  - tools/check-syntax -f && git diff && [ `git diff | wc -l` -eq 0 ]
```

Comment:
Why not the following?
```
  - tools/check-syntax -f && git diff --exit-code
```

Reply:
Good point, I didn't realize that git diff has such option. Thanks!